src/main/java/com/powsybl/openloadflow/ac/outerloop/AcloadFlowEngine.java

```diff
@@ -134,7 +135,7 @@ public AcLoadFlowResult run() {
 
     public AcLoadFlowResult run(Reporter reporter) {
         if (equationSystem == null) {
-            LOGGER.info("Start AC loadflow on network {}", network.getNum());
+            LOGGER.info("Start AC loadflow on network {}", network);
```

Comment:
What does this log indeed? We don't print any number here.

Reply:
Yes, we do, through the `LfNetwork::toString` method